msu/vanilla_mod/vanilla_keybinds.nut

```diff
@@ -260,29 +260,73 @@ local function isCampfireScreen()
 	}
 }, "Toggle Forced Attack", ::MSU.Key.KeyState.Release | ::MSU.Key.KeyState.Press);
 
-// World Continuous, doesn't work cuz we handle keybinds differently from vanilla ):
-
-// ::MSU.Vanilla.Keybinds.addSQKeybind("world_moveCamera_left", "left/a/q", ::MSU.Key.State.World, function()
-// {
-// 	if (::Settings.getTempGameplaySettings().CameraLocked)
-// 	{
-// 		this.m.WorldScreen.getTopbarOptionsModule().onCameraLockButtonPressed();
-// 	}
-
-// 	::World.getCamera().move(-1500.0 * ::Time.getDelta() * ::Math.maxf(1.0, ::World.getCamera().Zoom * 0.66), 0);
-// 	return true;
-// }, "Move Camera Up", null, ::MSU.Key.KeyState.Continuous | ::MSU.Key.KeyState.Press);
-
-// ::MSU.Vanilla.Keybinds.addSQKeybind("world_moveCamera_right", "right/d", ::MSU.Key.State.World, function()
-// {
-// 	if (::Settings.getTempGameplaySettings().CameraLocked)
-// 	{
-// 		this.m.WorldScreen.getTopbarOptionsModule().onCameraLockButtonPressed();
-// 	}
-
-// 	::World.getCamera().move(1500.0 * ::Time.getDelta() * ::Math.maxf(1.0, ::World.getCamera().Zoom * 0.66), 0);
-// 	return true;
-// }, "Move Camera Right", null, ::MSU.Key.KeyState.Continuous | ::MSU.Key.KeyState.Press);
+
+local function moveCameraWorld(_x, _y)
+{
+	if (::Settings.getTempGameplaySettings().CameraLocked)
+	{
+		this.m.WorldScreen.getTopbarOptionsModule().onCameraLockButtonPressed();
+	}
+
+	::World.getCamera().move(_x * ::Time.getDelta() * ::Math.maxf(1.0, ::World.getCamera().Zoom * 0.66), _y * ::Time.getDelta() * ::Math.maxf(1.0, ::World.getCamera().Zoom * 0.66));
+	return true;
+}
+//Camera Left
+
+::MSU.Vanilla.Keybinds.addSQKeybind("world_moveCamera_left", "left/a/q", ::MSU.Key.State.World, function()
+{
+	return moveCameraWorld(-1500, 0);
+}, "Move Camera Up", ::MSU.Key.KeyState.Continuous | ::MSU.Key.KeyState.Press).setCallContinuously(true);
+
+//Camera Right
+
+::MSU.Vanilla.Keybinds.addSQKeybind("world_moveCamera_right", "right/d", ::MSU.Key.State.World, function()
+{
+	return moveCameraWorld(1500, 0);
+}, "Move Camera Right", ::MSU.Key.KeyState.Continuous | ::MSU.Key.KeyState.Press).setCallContinuously(true);
+
+//Camera Up
+
+::MSU.Vanilla.Keybinds.addSQKeybind("world_moveCamera_up", "up/w/z", ::MSU.Key.State.World, function()
+{
+	return moveCameraWorld(0, 1500);
+}, "Move Camera Up", ::MSU.Key.KeyState.Continuous | ::MSU.Key.KeyState.Press).setCallContinuously(true);
+
+//Camera Down
+
+::MSU.Vanilla.Keybinds.addSQKeybind("world_moveCamera_down", "down/s", ::MSU.Key.State.World, function()
+{
+	return moveCameraWorld(0, -1500);
+}, "Move Camera Down", ::MSU.Key.KeyState.Continuous | ::MSU.Key.KeyState.Press).setCallContinuously(true);
+
+// Camera Top Left
+
+::MSU.Vanilla.Keybinds.addSQKeybind("world_moveCamera_leftup", "left+up/left+w/left+z/a+up/a+w/a+z/q+up/q+w/q+z", ::MSU.Key.State.World, function()
```

Comment:
Should it be camelCase in `leftup` in the id i.e. `leftUp`? Similarly for the other keybinds you've added.